_docs-sources/foundations/landing-zone/add-aws-account.md

```diff
@@ -6,46 +9,76 @@ This document provides instructions for provisioning a new AWS account using Gru
+  To initiate the process, you can use the GitHub Actions workflow in your `infrastructure-live` repo by following the below steps. Alternatively, you can choose to manually create the account request file and associated PR by following the steps in the **Account Request File (Manual)** dropdown below.
 
-:::
+<Tabs groupId="request-aws-account">
+<TabItem value="GitHub Action Workflow" label="GitHub Action Workflow" default>
+
+1. Navigate to your `infrastructure-live` repo and select the **Actions** tab.
+
+1. Select the **Account Factory** workflow.
+
+1. Select the **Run Workflow** dropdown menu and fill in the parameters.
+
+1. Select **Run Workflow** to kick off the process.
+
+The workflow will automatically create a PR to be reviewed.
+
+</TabItem>
+<TabItem value="Account Request File (Manual)" label="Account Request File (Manual)">
+
+1. To initiate the process, create an `account-<AWS-ACCOUNT-NAME>.yml` file in the `_new-account-requests` folder located in the root of your `infrastructure-live` repository. This file will be used to create a pull request and add the new account to your organization. The file should have the following format:
+
+  ```yaml account-<AWS-ACCOUNT-NAME>.yml
+  account_name: <AWS-ACCOUNT_NAME>
+  account_email: <ROOT-EMAIL-FOR-ACCOUNT>
+  organizational_unit_name: <OU-TO-ADD-ACCOUNT-TO>
+  aws_region: <REGION-T0-DEPLOY-BASELINES>
+  org_name_prefix: <ORG-NAME-PREFIX-FOR-RESOURCES>
+  sso_user_first_name: <SSO-USER-FIRST-NAME>
+  sso_user_last_name: <SSO-USER-LAST-NAME>
+  sso_user_email: <SSO-USER-EMAIL>
+  account_baseline_modules_version: <ACCOUNT-BASELINE-MODULES-VERSION>
+  requested_by: <GITHUB_USER_ID_OR_EMAIL>
+  ```
+
+    - `account_name`: The name of the account in AWS.
+    - `account_email`: The email address to associate with the account.
+    - `organizational_unit_name`: The name of the Organizational Unit(OU) where the new account will be added. Ensure this exactly matches an existing OU name in your organization, for example, “Security”.
+    - `aws_region`: The AWS region where the Gruntwork account baselines will be deployed. e.g., `us-east-1`.
+    - `org_name_prefix`: The prefix to use for some resources created by Gruntwork modules. For instance, S3 bucket files will be prefixed with this value
+    - `sso_user_first_name`: The first name of the user to create in AWS IAM Identity Center (Successor to AWS Single Sign-On(SSO)).
+    - `sso_user_last_name`: The last name of the user to create in AWS IAM Identity Center.
+    - `sso_user_email`: The email address of the user to create in AWS IAM Identity Center. This user will be able to login to the new account using AWS IAM Identity Center.
+    - `account_baseline_modules_version`: The version of the your account baseline modules to use for the new account. e.g., `v0.0.1`
+    - `requested_by`: The GitHub user ID or email address of the entity requesting the new account, for audit purposes.
```

Comment:
I recommend pulling this list of parameters out of the tabs themselves since they apply to both approaches.

Reply:
Pulled this out and added it before the tabbed sections. Let me know what you think!

Reply:
Looks great!

I'm noticing minor nits now that I look more closely, but they are with the original content and I think we've held this PR up long enough! I'll approve it as is and use the "edit this page" feature to make a few typo fixes.